src/forms/GalleryPreview.jsx

```diff
@@ -12,10 +12,12 @@ export default class GalleryPreview extends React.Component {
   }
 
   render() {
+    const gallery = this.props[this.props.activeGallery];
+    if(!gallery) return null;
```

Comment:
spacing after our `if` statements is the only thing that separates us from the animals